src/web/src/components/Posts/GitHubInfo/Post.tsx

```diff
@@ -16,16 +16,16 @@ import {
 } from '@material-ui/core';
 import ErrorRoundedIcon from '@material-ui/icons/ErrorRounded';
 import LiteYouTubeEmbed from 'react-lite-youtube-embed';
-import GithubInfoProvider from '../GithubInfoProvider';
-import { Post } from '../../interfaces';
-import AdminButtons from '../AdminButtons';
-import Spinner from '../Spinner';
-import PostDesktopInfo from './PostInfo';
-import PostAvatar from './PostAvatar';
+import GithubInfoProvider from '../../GithubInfoProvider';
+import { Post } from '../../../interfaces';
```

Comment:
nit: Organize imports to components first and interfaces after?

Reply:
I reorganize the imports.